__tests__/app.test.js

```diff
@@ -133,8 +134,8 @@ describe('GET /api/reviews/:review_id/comments', () => {
         return request(app).get('/api/reviews/10000/comments')
         .expect(404)
         .then((result) => {
-            const errMsg = result.body.msg
-            expect(errMsg).toBe('The id number 10000, is not currently in use')
+            const errorMessage = result.body.msg
+            expect(errorMessage).toBe('The id number 10000, is not currently in use')
```

Comment:
This is a good test but I would caution against overly specific error messages. They can unintentionally give bad actors info about your data. It's not a big deal here obviously as our database is available to see, but just something to keep in mind